xarray/core/dataset.py

```diff
@@ -2416,6 +2416,66 @@ def from_dataframe(cls, dataframe):
             obj[name] = (dims, data)
         return obj
 
+    def to_dask_dataframe(self, set_index=False):
+        """
+        Convert this dataset into a dask.dataframe.DataFrame.
+
+        Both the coordinate and data variables in this dataset form
+        the columns of the DataFrame.
+
+        If set_index=True, the dask DataFrame is indexed by this dataset's
+        coordinate.  Since dask DataFrames to not support multi-indexes,
+        set_index only works if there is one coordinate dimension.
+        """
+
+        import dask.dataframe as dd
+
+        ordered_dims = self.dims
```

Comment:
We should probably add that `dims_order` keyword argument. Then this becomes something like:

```python
if dims_order is None:
    dims_order = self.dims
ordered_dims = OrderedDict((k, self.dims[k]) for k in dims_order)
```